### PR TITLE
Pandas 0.12

### PR DESCRIPTION
**Pull request by pforai**
This adds pandas 0.12 which now has external daps. Deps are included in goolf and ictce flavors.

### CODE REVIEW

**Comment by boegel** [comment]
@pforai: did you chuck the branch for this?

File: easybuild/easyconfigs/p/pandas/pandas-0.12.0-goolf-1.4.10-Python-2.7.3.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+source_urls = ['http://pypi.python.org/packages/source/p/%(name)s/']
+sources = [SOURCE_TAR_GZ]
+
+python = "Python"
+pythonversion = "2.7.3"
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [(python, pythonversion),
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pandas/pandas-0.12.0-goolf-1.4.10-Python-2.7.3.eb]: do the indenting like all other easyconfigs

File: easybuild/easyconfigs/p/pandas/pandas-0.12.0-goolf-1.4.10-Python-2.7.3.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+name = "pandas"
+version = "0.12.0"
+
+homepage = "http://pypi.python.org/pypi/pandas/"
+description = """pandas is an open source, BSD-licensed library providing high-performance, easy-to-use data structures
+and data analysis tools for the Python programming language."""
+
+toolchain = {'name': 'ictce', 'version': '5.3.0'}
+
+source_urls = ['http://pypi.python.org/packages/source/p/%(name)s/']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pandas/pandas-0.12.0-goolf-1.4.10-Python-2.7.3.eb]: use `PYPI_SOURCE`

File: easybuild/easyconfigs/p/pydateutil/pydate-util-2.2-goolf-1.4.10-Python-2.7.3.eb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+easyblock = "PythonPackage"
+
+name = 'python-dateutil'
+version = '2.2'
+
+homepage = 'https://pypi.python.org/pypi/python-dateutil'
+description = """The dateutil module provides powerful extensions to the datetime module available in the Python standard library. """
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pydateutil/pydate-util-2.2-goolf-1.4.10-Python-2.7.3.eb]: delete needless empty lines in this files and clean up the indenting

File: easybuild/easyconfigs/p/pydateutil/pydate-util-2.2-goolf-1.4.10-Python-2.7.3.eb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+majorversion = "%s.x" % ".".join(version.split('.')[:-1])
+
+
+source_urls = ['https://pypi.python.org/packages/source/p/python-dateutil/']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pydateutil/pydate-util-2.2-goolf-1.4.10-Python-2.7.3.eb]: `PYPI_SOURCE`

File: easybuild/easyconfigs/p/pydateutil/pydate-util-2.2-goolf-1.4.10-Python-2.7.3.eb [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+sources = ['%(name)s-%(version)s.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pydateutil/pydate-util-2.2-goolf-1.4.10-Python-2.7.3.eb]: `SOURCE_TAR_GZ`

File: easybuild/easyconfigs/p/pytz/pytz-2013.8-goolf-1.4.10-Python-2.7.3.eb [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+homepage = 'https://code.google.com/p/pytz/'
+description = """World timezone definitions, modern and historical."""
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+
+majorversion = "%s.x" % ".".join(version.split('.')[:-1])
+
+
+source_urls = ['https://pypi.python.org/packages/source/p/pytz/']
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pytz/pytz-2013.8-goolf-1.4.10-Python-2.7.3.eb]: same as above

File: easybuild/easyconfigs/p/pytz/pytz-2013.8-ictce-5.3.0-Python-2.7.3.eb [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+name = 'pytz'
+version = '2013.8'
+
+homepage = 'https://code.google.com/p/pytz/'
+description = """World timezone definitions, modern and historical."""
+toolchain = {'name': 'ictce', 'version': '5.3.0'}
+
+
+majorversion = "%s.x" % ".".join(version.split('.')[:-1])
```
Comment (wpoely86) [inline easybuild/easyconfigs/p/pytz/pytz-2013.8-ictce-5.3.0-Python-2.7.3.eb]: unneeded?

File: easybuild/easyconfigs/s/six/six-1.4.1-goolf-1.4.10-Python-2.7.3.eb [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+version = '1.4.1'
+
+homepage = 'https://pypi.python.org/pypi/six'
+description = """Six provides simple utilities for wrapping over differences between Python 2 and Python 3. 
+It is intended to support codebases that work on both Python 2 and 3 without modification. 
+six consists of only one Python file, so it is painless to copy into a project."""
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+
+majorversion = "%s.x" % ".".join(version.split('.')[:-1])
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/six/six-1.4.1-goolf-1.4.10-Python-2.7.3.eb]: unneeded?

File: easybuild/easyconfigs/s/six/six-1.4.1-goolf-1.4.10-Python-2.7.3.eb [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+
+
+source_urls = ['https://pypi.python.org/packages/source/s/six/']
+sources = ['%(name)s-%(version)s.tar.gz']
```
Comment (wpoely86) [inline easybuild/easyconfigs/s/six/six-1.4.1-goolf-1.4.10-Python-2.7.3.eb]: same

**Comment by wpoely86** [comment]
some style remarks, else lgtm

**Comment by wpoely86** [comment]
and a rebase/merge with develop

**Comment by boegel** [comment]
@wpoely86: can you send @pforai a PR to fix these issues?

**Comment by verdurin** [comment]
@boegel: I'd be happy to do that, but I'm not sure how to do a PR against this? Can't see where it's pushed from.

**Comment by boegel** [comment]
I think @pforai must have chucked his branch (or the entire repo??)...

Since we already have several more recent easyconfigs for pandas, this can just be closed? @pforai!

**Comment by pforai** [comment]
yeah, throw his away :)

**Pull request closed without merging**